.storybook/main.ts

```diff
@@ -24,44 +17,10 @@ const config: StorybookConfig = {
     },
   ],
   framework: {
-    name: '@storybook/react-webpack5',
-    options: {
-      fastRefresh: true,
-    },
+    name: 'storybook-react-rspack',
   },
   docs: {
     autodocs: 'tag',
   },
-  webpackFinal: async config => {
```

Comment:
@th3fallen do you think that its worth trying to tackle rspack with storybook right now?

Otherwise if we just fix jest then this is basically ready to go.

And we'd just need to use webpack for a little while for storybook only.